cmd/kubeadm/app/phases/bootstraptoken/clusterinfo/clusterinfo_test.go

```diff
@@ -120,7 +83,7 @@ func TestCreateBootstrapConfigMapIfNotExists(t *testing.T) {
 			})
 		}
 
-		err = CreateBootstrapConfigMapIfNotExists(client, file.Name())
+		err = createBootstrapConfigMapIfNotExists(client, file.Name())
```

Comment:
Just commit weirdness. It changed (wrongly) in the first commit and then reverted in the second.
Didn't squash properly it seems. Now fixed.